app/controllers/hub/clients_controller.rb

```diff
@@ -98,7 +98,7 @@ def edit_take_action
         @client,
         current_user,
         tax_return_id: params.dig(:tax_return, :id)&.to_i,
-        current_state: params.dig(:tax_return, :current_state),
+        status: params.dig(:tax_return, :current_state) || params.dig(:tax_return, :status),
```

Comment:
Because the field is called current_state on the tax return drop down in the client header, we want to load from current_state there. But when we change on the page, we need to call it status.